### DISCUSSION TITLE
Expose TAA’s motion vectors as a uniform hint in GDshader

### DISCUSSION THREAD

**Discussion by catboy-catfish** [godotengine/godot-proposals#9506] [3D]
Would be cool if you could expose the motion vectors (for TAA) as a uniform hint in the GDshader language (if it's not exposed in RenderingEffects already.) This would make it possible to add motion blur as a shader effect.

`uniform sampler2D motionvectors: hint_motvec_texture`

**Comment by AThousandShips** [comment]
See also:
* https://github.com/godotengine/godot-proposals/issues/5888

**Reply by catboy-catfish** [reply]
Closing this thread because it's a duplicate of the PR mentioned
